vendor/github.com/pkg/errors/errors.go

```diff
@@ -174,6 +177,18 @@ func (w *withStack) Format(s fmt.State, verb rune) {
 	}
 }
 
+// ToStatus implements the libkb.ExportableError interface. That way when we
+// wrap exportable errors, they're still exportable.
+func (w *withStack) ToStatus() (s keybase1.Status) {
+	if exportable, ok := w.error.(libkb.ExportableError); ok {
+		return exportable.ToStatus()
+	}
+	s.Code = int(keybase1.StatusCode_SCGeneric)
+	s.Name = "GENERIC"
+	s.Desc = w.Error()
+	return
+}
+
```

Comment:
Woops, I made this change in a vendored package. I'll make it upstream and then revendor it here.

Reply:
Err, hell, that's an upstream package. I'll figure something out...

Reply:
(Fixed this by taking a dependency on `pkg/errors` in client.)